preface-2e.qmd

```diff
@@ -1,31 +1,31 @@
+Boas vindas à segunda edição do livro "R para Ciência de Dados"!
+Esta é uma revisão importante da primeira edição, removendo material que não consideramos mais útil, adicionando material que desejamos incluir na primeira edição e atualizando o texto e o código de acordo com as melhores práticas.
+Também estamos muito animados em dar as boas-vindas a uma nova co-autora: Mine Çetinkaya-Rundel, uma renomada educadora em ciência de dados e uma de nossas colegas na Posit (antiga RStudio).
 
-A brief summary of the biggest changes follows:
+Segue abaixo um breve resumo das maiores mudanças:
 
--   The first part of the book has been renamed to "Whole game".
-    The goal of this section is to give you the rough details of the "whole game" of data science before we dive into the details.
+-   A primeira parte do livro foi renomeada para "Jogo completo".
+    O objetivo desta seção é fornecer a você os detalhes aproximados do "jogo completo" da ciência de dados antes de nos aprofundarmos nos detalhes.
 
--   The second part of the book is "Visualize".
-    This part gives data visualization tools and best practices a more thorough coverage compared to the first edition.
-    The best place to get all the details is still the [ggplot2 book](http://ggplot2-book.org/), but now R4DS covers more of the most important techniques.
+-   A segunda parte do livro é chamada "Visualizar".
+    Esta parte oferece uma cobertura mais abrangente das ferramentas e práticas recomendadas de visualização de dados em comparação à primeira edição.
+    O melhor lugar para obter todos os detalhes ainda é o [livro do ggplot2](http://ggplot2-book.org/), mas agora o R4DS aborda de forma mais abrangente as técnicas importantes.
 
--   The third part of the book is now called "Transform" and gains new chapters on numbers, logical vectors, and missing values.
-    These were previously parts of the data transformation chapter, but needed much more room to cover all the details.
+-   A terceira parte do livro agora é chamada de "Transformar" e inclui novos capítulos sobre números, vetores lógicos e valores faltantes (*missing values* - *NA*).
+    Esses tópicos eram anteriormente parte do capítulo de transformação de dados, mas precisavam de mais espaço para cobrir todos os detalhes.
 
--   The fourth part of the book is called "Import".
-    It's a new set of chapters that goes beyond reading flat text files to working with spreadsheets, getting data out of databases, working with big data, rectangling hierarchical data, and scraping data from web sites.
+-   A quarta parte do livro é chamada de "Importar".
+    São novos capítulos que vão além da leitura de arquivos de texto simples para trabalhar com planilhas, importar dados de bancos de dados (*databases*), trabalhar com grandes conjuntos de dados (*big data*), dados retangulares hierárquicos e extrair dados de sites da web (*web scraping*).
```

Comment:
Eu acho que `dados retangulares hierárquicos` não é a melhor tradução, já que é no sentido de "tornar dados hierárquicos em dados retangulares". O início do capítulo [Hierarchical data](https://r4ds.hadley.nz/rectangling) explica melhor o propósito.

Reply:
eu tinha achado confuso inicialmente, mas com a sugestão do @nextmarte realmente faz mais sentido! vou mudar, obrigada!